diff --git a/chrome/common/importer/importer_bridge.h b/chrome/common/importer/importer_bridge.h
index b4250c91d1b83ea920b3de9cd6b1a7929b30ffc5..7a4af36c97e31e64192e95a29600bc7685ead908 100644
--- a/chrome/common/importer/importer_bridge.h
+++ b/chrome/common/importer/importer_bridge.h
@@ -15,10 +15,13 @@
 #include "chrome/common/importer/importer_data_types.h"
 #include "chrome/common/importer/importer_url_row.h"
 #include "components/favicon_base/favicon_usage_data.h"
+#include "net/cookies/canonical_cookie.h"
 
 class GURL;
 struct ImportedBookmarkEntry;
 struct ImporterAutofillFormDataEntry;
+struct BraveStats;
+struct BraveReferral;
 
 namespace autofill {
 struct PasswordForm;
@@ -58,6 +61,15 @@ class ImporterBridge : public base::RefCountedThreadSafe<ImporterBridge> {
   virtual void SetAutofillFormData(
       const std::vector<ImporterAutofillFormDataEntry>& entries) = 0;
 
+  virtual void SetCookies(
+      const std::vector<net::CanonicalCookie>& cookies) {};
+
+  virtual void UpdateStats(
+      const BraveStats& stats) {};
+
+  virtual void UpdateReferral(
+      const BraveReferral& referral) {};
+
   // Notifies the coordinator that the import operation has begun.
   virtual void NotifyStarted() = 0;
 
